scripts/build-depends.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
+from email.parser import Parser
```

Comment:
I must say, this looks surprising, would you add a comment to explain why we're using this module?

Reply:
some time ago there was a specific rfc822 module, but that was moved to email.   https://docs.python.org/2/library/rfc822.html
Added a similar comment.